desktop/core/src/desktop/js/reactComponents/AlertComponent/AlertComponent.scss

```diff
@@ -0,0 +1,84 @@
+// Licensed to Cloudera, Inc. under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// 'License'); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@import '../../components/styles/variables';
+
+.flash-messages {
+    position: fixed;
+    width: 30%;
+    z-index: 9999;
+    top: 7px;
+    right: 0;
+    margin-right: 24px;
+    display: flex;
+    flex-flow: column;
+    row-gap: 10px;
+
+    button:hover,
+    button:focus {
+        background-color: unset;
+    }
+
+    button span {
+        font-size: 12px;
+        padding-top: 5px;
+    }
+
+    button svg.cdp-icon-close {
+        height: 16px;
+    }
+
+    .ant-alert-warning {
```

Comment:
there are styles for warning and success alerts, but the actual component is hardcode to type="error", why?

Reply:
Didn't got you.

Reply:
The AlertComponent you have written is hardcoded to use type="error".
https://github.com/cloudera/hue/pull/3488/files/9f37cc8cb4ec012b5495dcfcfac12b38dd325cbe#diff-774c7bd2d92554254689d35b3f2802b2440e5a2d5e95322e7286d30b9bceda97R49

Reply:
We are initially only supporting errors but we will add support for info and warning messages in the next iteration.